.github/workflows/deploy-to-test.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy to Test Environment
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  VAULT_PW: ${{ secrets.VAULT_PW }}
+
+permissions:
+    id-token: write   # This is required for requesting the JWT
+    contents: read    # This is required for actions/checkout
+
+jobs:
+  deploy-to-test:
+    name: Deploy to Test Environment
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Decrypt secrets
+        run: |
+          echo $VAULT_PW > .vault_password
+          bash ops/secrets --decrypt
+          mv -fv shared_files/encrypted/* shared_files/decrypted/
```

Comment:
We might be able to switch out this secrets step to use AWS Parameter Store instead.